prac2-pages/Simranpreet Kaur/login_page.dart

```diff
@@ -0,0 +1,181 @@
+// ignore_for_file: prefer_const_literals_to_create_immutables, prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'login2.dart' ;
+
+class LoginPage extends StatefulWidget{
+  const LoginPage({Key? key}) : super(key:key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+
+}
+class _LoginPageState extends State<LoginPage> {
+  bool check1 = true;
+  @override
+  Widget build(BuildContext context){
+    return Scaffold(
```

Comment:
Whenever the keypad pops up, when we click on a textfield, there occurs an error of Bottom Overflow. :exclamation: 
Kindly fix this critical issue in the subsequent commits. :thought_balloon: 

**Screenshot**:
![WhatsApp Image 2022-12-23 at 12 30 17](https://user-images.githubusercontent.com/106133014/209287999-62e1eaf5-a81c-4c3c-a503-0a033f1dab17.jpeg)

Reply:
Resolved! :+1: